### DISCUSSION TITLE
SwatchifyPro Theme Released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#3949] [Show and tell]
All the delights of Bootswatch in Oqtane plus customizable Admin and Main Container widths. Contains a Theme layout common in Oqtane plus a Dashboard layout. Simply select your layout and in the Themes Settings, select your desired theme and Admin and Content widths. Settings can be set for the Site or for the Page.

![image](https://github.com/oqtane/oqtane.framework/assets/9447612/c56eaf91-1cc6-4e1c-a17e-ab7f8e88ffdc)

![image](https://github.com/oqtane/oqtane.framework/assets/9447612/617a4a2c-779c-4b42-bcee-fe2df2831267)

**Comment by W6HBR** [comment]
@leigh-pointer  Is this available for download?

**Reply by leigh-pointer** [reply]
@W6HBR yes it is in the market place!

**Comment by W6HBR** [comment]
@leigh-pointer 
I guess I'm missing something here.  I see it listed in the marketplace, but all that seems to do is link to your website.  I can't find the download on your website either.

**Comment by leigh-pointer** [comment]
@W6HBR Its there: You will need to access it through the Themes Manager in your Oqtane instance.

![image](https://github.com/oqtane/oqtane.framework/assets/9447612/2379f9bd-dca7-4581-a800-8339ba02020b)
